org.eclipse.scanning.device.ui/src/org/eclipse/scanning/device/ui/device/scannable/ControlValueLabelProvider.java

```diff
@@ -115,6 +115,7 @@ public void positionPerformed(PositionEvent evt) throws ScanningException {
 	}
 	
 	private void updatePosition(PositionEvent evt) {
+		if (viewer.getViewer().isCellEditorActive()) return;

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00122)